openmetadata-ui/src/main/resources/ui/playwright/utils/serviceIngestion.ts

```diff
@@ -120,10 +120,7 @@ export const deleteService = async (
   await deleteResponse;
 
   // Closing the toast notification
-  await toastNotification(
-    page,
-    `Delete operation initiated for ${serviceName}`
-  );
+  await toastNotification(page, /deleted successfully!/, 5 * 60 * 1000); // Wait for up to 5 minutes for the toast notification to appear
```

Comment:
this will make a test timeout before this is executed.

Reply:
since the async deletion with take sometime to resolve, have used this timer. Even though will monitor the PR's and AUT result to track the accurate time for this resolve and update the Timer with Plus seconds in it in further PR.